packages/@aws-cdk/cfn-include/README.md

```diff
@@ -0,0 +1,184 @@
+# Include CloudFormation templates in the CDK
+
+<!--BEGIN STABILITY BANNER-->
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+This module contains a set of classes whose goal is to facilitate working
+with existing CloudFormation templates in the CDK.
+It can be thought of as an extension of the capabilities of the
+[`CfnInclude` class](../@aws-cdk/core/lib/cfn-include.ts).
+
+## Basic usage
+
+Let's say you have the file `my-template.json` with the following contents:
+
+```json
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": "some-bucket-name"
+      }
+    }
+  }
+}
+```
+
+You can include it in your CDK application using the following code:
+
+```typescript
+import * as inc from '@aws-cdk/cfn-include';
+
+const cfnTemplate = new inc.CdkInclude(this, 'Template', {
```

Comment:
is it intentional that the module is named `cfn-include` but the entry point of this module is `CdkInclude`. 

Intentional? Maybe make this consistent?

Reply:
I think it should be called `CfnInclude`

Reply:
What about the name `@aws-cdk/cfn-import` for the package, and `CfnImport` for the class, to differentiate between this, and the existing `CfnInclude` class?

Thoughts on this @eladb @nija-at ?

Reply:
I still don’t understand why do we need to differentiate between them... This new class is a superset of the existing one. It should have full feature parity with what we have in core _and_ offer additional capabilities.

Reply:
Yes, looks like `@aws-cdk/cfn-include` doesn't play well with monocdk - from the build logs:

```
monocdk-experiment: [96mcore/lib/cfn-include.ts[0m:[93m17[0m:[93m1[0m - [91merror[0m[90m TS9999: [0mJSII: Submodule "cfn_include" conflicts with "CfnInclude". Restricted names are: cfn_include, cfnInclude, CfnInclude
```

Reply:
Ha. Interesting. It’s because we hoist all the types from the core module to the root, so now the submodule conflicts with the original class name. Good job @RomainMuller!

We should find a creative alternative for the submodule name. I wonder if this module might include other types that will assist in migration from CloudFormation, so perhaps something like @aws-cdk/migration?

Reply:
How about `@aws-cdk/cloudformation-include`?

Reply:
I went with `@aws-cdk/cloudformation-include` for the package name and `CfnInclude` for the class name. Let me know if you disagree.